### PR TITLE
Implement Window#top.

### PR DESCRIPTION
**Pull request by Ms2ger**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/6600)

### CODE REVIEW

File: components/script/dom/window.rs [hunk 1]
```diff
@@ -442,6 +442,15 @@ impl<'a> WindowMethods for &'a Window {
         self.parent().unwrap_or(self.Window())
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-parent
```
Comment (Manishearth) [inline components/script/dom/window.rs]: https://html.spec.whatwg.org/multipage/browsers.html#dom-top

**Comment by jdm** [comment]
```
/old-tests/submission/Opera/script_scheduling/034.html
------------------------------------------------------
PASS expected FAIL  scheduler: innerHTML adding frames with JS in
```

**Pull request merged**